### PR TITLE
Documentation improvements (`extractors/excel`)

### PR DESCRIPTION
**Pull request by Cerebrovinny**

### CODE REVIEW

File: bw2io/extractors/excel.py [hunk 1]
```diff
@@ -67,21 +106,31 @@ def extract(cls, filepath):
...
         """
-        Extract data from a single sheet in an Excel file.
+        Extract data from a single sheet in an Excel workbook.
 
         Parameters
         ----------
         wb : openpyxl.workbook.Workbook
             The workbook object with the sheet to extract data from.
-        name : str
+        sheetname : str
```
Comment (michaelweinold) [inline bw2io/extractors/excel.py]: Not sure you can change the parameter name here, if the parameter name in the function definition is not changed.

File: bw2io/extractors/excel.py [hunk 2]
```diff
@@ -67,21 +106,31 @@ def extract(cls, filepath):
...
+            A list of lists containing the data from the sheet.
+
+        Notes
+        -----
+        This method is called by the 'extract' method to extract the data from each sheet in the workbook.
+
+        Examples
+        --------
+        >>> wb = openpyxl.load_workbook('example.xlsx')
+        >>> sheetname = 'Sheet1'
```
Comment (michaelweinold) [inline bw2io/extractors/excel.py]: ...compare comment above.

File: bw2io/extractors/excel.py (hunk 1 above)
Comment (Cerebrovinny) [reply bw2io/extractors/excel.py]: resolved

File: bw2io/extractors/excel.py (hunk 2 above)
Comment (Cerebrovinny) [reply bw2io/extractors/excel.py]: resolved

File: bw2io/extractors/excel.py [hunk 3]
```diff
@@ -5,17 +5,27 @@
...
     Parameters
     ----------
-    cell : openpyxl.cell.cell.cell
-        The cell to retrieve the value from.
+    cell : openpyxl.cell.cell.Cell
+        The cell to get the value from.
 
-    Returns
-    -------
+    Returns:
```
Comment (michaelweinold) [inline bw2io/extractors/excel.py:15]: @Cerebrovinny, something here broke the correct rendering of the documentation. Maybe the "`:`"?

Screenshot 2023-04-18 at 12 07 56

Please fix this in a new PR.

Comment (Cerebrovinny) [reply bw2io/extractors/excel.py:15]: fixed

**Pull request merged**